src/sass/rivet.scss

```diff
@@ -24,6 +24,7 @@
 @import "components/buttons-segmented";
 @import "components/checkboxes";
 @import "components/dropdown";
+@import "components/features";
```

Comment:
Same as the comment above. Let's make all component references singular (`/feature`) to stay consistent.